peps/pep-0729.rst

```diff
@@ -79,6 +79,9 @@ Some concrete examples include:
   type system features. Type checkers are expected to treat symbols in this
   module the same as symbols in :mod:`typing`, but this is not explicitly
   specified in any of the PEPs.
+* Various semantics are informally agreed upon by type checkers, but not
```

Comment:
This maybe fits better in the next section? More examples would definitely be good here.

Also, we need ReST links, I'll add suggestions for that.